### PR TITLE
Fix Drone Env Variable

### PR DESCRIPTION
**Pull request by maplain**
Use $ to reference env variable instead of $$

It's uploading something like: 
  vic_10BUILD_NUMBER.tar.gz
  vic_9BUILD_NUMBER.tar.gz

### CODE REVIEW

File: .drone.nightly.yml [hunk 1]
```diff
@@ -4,37 +4,37 @@ clone:
 
 build:
   integration-test:
-    image: $${TEST_BUILD_IMAGE=gcr.io/eminent-nation-87317/vic-integration-test:1.30}
+    image: ${TEST_BUILD_IMAGE=gcr.io/eminent-nation-87317/vic-integration-test:1.30}
```
Comment (jakedsouza) [inline .drone.nightly.yml]: I dont think we are updating the nightly builds to 05 yet. @mhagen-vmware can provide details

File: .drone.ova.yml [hunk 2]
```diff
@@ -15,7 +15,7 @@ clone:
 
 build:
   unified-ova:
-    image: $${TEST_BUILD_IMAGE=gcr.io/eminent-nation-87317/vic-integration-test:1.30}
+    image: ${TEST_BUILD_IMAGE=gcr.io/eminent-nation-87317/vic-integration-test:1.30}
```
Comment (jakedsouza) [inline .drone.ova.yml]: same here, no change yet to drone 05

File: .drone.nightly.yml (hunk 1 above)
Comment (mhagen-vmware) [inline .drone.nightly.yml]: we were not intending too... does it not work anymore if we don't upgrade nightly?

File: .drone.ova.yml (hunk 2 above)
Comment (mhagen-vmware) [inline .drone.ova.yml]: correct

File: .drone.yml [hunk 3]
```diff
@@ -176,7 +176,6 @@ pipeline:
   report-coverage:
     image: plugins/coverage
     server: https://coverage.vmware.run
-    token: ${GITHUB_AUTOMATION_API_KEY}
```
Comment (mhagen-vmware) [inline .drone.yml]: I think we need this to avoid hitting the non-token limit, which is very strict?

Comment (maplain) [inline .drone.yml]: @jakedsouza

Comment (jakedsouza) [inline .drone.yml]: The plugin accepts a secret env var `GITHUB_TOKEN` from drone, it is still being passed in, not via the yml but via drone .
As per their docs https://github.com/drone-plugins/drone-coverage

Comment (jakedsouza) [inline .drone.yml]: I pushed the corresponding secret to drone, will push to our internal repo once this PR is in

**Pull request merged**